ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -123,11 +123,23 @@ message DeleteUserResponse {
 
 // Required authorization: ``HasRight(ParticipantAdmin)``
 message ListUsersRequest {
+
+    // Pagination token to determine the specific page to fetch.
+    string page_token = 2; // TODO:  url-safe base64 encoded protocol buffer,
+
+
+    // Maximum number of results to be returned by the server. Server will return no more than that many results
```

Comment:
```suggestion
    // Maximum number of results to be returned by the server. The server will return no more than that many results, but it might return fewer.
```

Reply:
Ok